playbilling/src/test/java/com/google/androidbrowserhelper/playbilling/provider/MethodDataTest.java

```diff
@@ -1,3 +1,17 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
nit: 2021

```suggestion
// Copyright 2021 Google Inc. All Rights Reserved.
```

Reply:
Technically the file is from 2020, I just forgot to add the copyright until 2021.